### ISSUE TITLE
Tutorial on how to use the catalog

### ISSUE THREAD

**Issue by dgarijo** [mintproject/MINT-ModelCatalogExplorer#101]
It's missing

**Comment by yashdholakia113** [comment]
Hey Daniel, does that tutoial exist or I also create a tutorial on how to use the ModelCatalogExplorer?

**Comment by dgarijo** [comment]
This tutorial does not yet exist :)

**Comment by yashdholakia113** [comment]
Is a youtube video using screen recording fine ?

**Comment by dgarijo** [comment]
No, we should use text and screenshots. But among all urgent issues, this is the last one we should be addressing. It's more important to fix the functionality issues at the moment.

**Comment by yashdholakia113** [comment]
Okay!!

**Comment by dgarijo** [comment]
I am assigning myself to work on this with @yashdholakia113
